packages/client/tests/unit.vercelignore.test.ts

```diff
@@ -16,7 +17,9 @@ describe('Test `getVercelIgnore()`', () => {
     try {
       await getVercelIgnore(fixture);
     } catch (_err) {
-      err = _err;
+      if (_err instanceof Error) {
+        err = _err;
+      }
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (non-blocking):** ‏check else case

If for some reason `_err` is not an `Error`, we should make sure the error shows what it was. Right now, if this test fails for this reason, the failure we'll get is that `err` is `null`, which isn't not super helpful.

I suggest:

```suggestion
      if (!(_err instanceof Error)) {
        throw new Error(`Error "${_err}" not instanceof Error`);
      } 
      
      err = _err;
```